product_supplierinfo_for_customer_sale_quick/views/product_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="product_tree_view" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="sale_quick.product_tree_view4sale" />
+        <field name="arch" type="xml">
+            <field name="display_name" position="after">
+                <field name="product_customer_code" optional="show" />
+                <field name="product_customer_name" optional="show" />
+            </field>
+
+        </field>
+    </record>
+
+    <record id="product_search_form_view" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field
+            name="inherit_id"
+            ref="base_product_mass_addition.product_search_form_view"
+        />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="product_customer_name" />
+                <field name="product_customer_code" />
+            </field>
+        </field>
+    </record>
+</odoo>
```

Comment:
Continuing from [here](https://github.com/OCA/sale-workflow/pull/2775#discussion_r1502983285) since Github won't let me reply directly for some reason

This is ok now, but I don't think you can see those filters, right?

I think in [sale_quick](https://github.com/OCA/sale-workflow/blob/14.0/sale_quick/models/sale_order.py#L42) you need to specify the use of this search view, just like it was done in [purchase_quick](https://github.com/OCA/purchase-workflow/blob/14.0/purchase_quick/models/purchase_order.py#L44-L46)